plugins/outputs/influxdb_v2/influxdb.go

```diff
@@ -32,12 +32,15 @@ var sampleConfig = `
   ## Token for authentication.
   token = ""
 
-  ## Organization is the name of the organization you wish to write to.
+  ## Organization is the name of the organization you wish to write to; must exist.
   organization = ""
 
   ## Bucket to the name fo the bucketwrite into; must exist.
   bucket = ""
 
+  ## Precision for the unix timestamps within the body line-protocol.
+  # precision = "ns"
```

Comment:
I don't think telegraf uses nanoseconds, does it?

Reply:
It does, though rounding is enabled by default on non-service inputs.  I think here we should do the same here as we do in the InfluxDB v1 output, and always send in nanosecond without the option to change it.

Reply:
nanoseconds is the default in the platform spec, I'll remove the option to change it